helm-chart/templates/nextcloud-aio-onlyoffice-persistentvolumeclaim.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   storageClassName: {{ .Values.STORAGE_CLASS }}
   {{- end }}
   accessModes:
-    - ReadWriteMany
+    - ReadWriteOnce
   resources:
     requests:
-      storage: {{ .Values.MAX_STORAGE_SIZE }}
```

Comment:
It looks like the replacement doesn't work here anymore?

Reply:
i initially didnt intend to make all volumes configurable but found a solution with https://github.com/nextcloud/all-in-one/pull/1848/commits/b6c3a777b0628cf3116b045bbade24e440d04eb9

Reply:
Improved with https://github.com/nextcloud/all-in-one/pull/1848/commits/133b149ee3272d6ac7815b3230edc5fd7215617a